docs/menu.yml

```
@@ -38,6 +38,8 @@ catalog:
             path: "/en/papers/stam"
           - name: "SDK Profiling to Fix the Blind Spot"
             path: "/en/concepts-and-designs/sdk-profiling"
+          - name: "EBPF CPU Profiling to pinpoint Service Mesh Critical Performance Impact"
```

```suggestion
          - name: "eBPF CPU Profiling to pinpoint Service Mesh Critical Performance Impact"
```

Is lower case e better?